### DISCUSSION TITLE
Single stack from CloudFormation with multiple agents when we have an installation with more than one server

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#297] [Feature Requests]
is there a way to have a single stack from CloudFormation with multiple agents when we have an installation with more than one server?

**Comment by drupaladmin** [comment]
Looks like we don't have such single stack.


Interesting, am I right that you mean one CloudFormation stack to install agent for ex. for 5 servers at the same time? To save time on installing 5 different stacks

**Reply by drupaladmin** [reply]
correct! time and $$ if any
if not.. I'll probably get that in Kubernetes

**Comment by jtomaszon** [comment]
Having different stacks just because we have multiple servers makes no sense.
Options that we will love to see:
- Configuring one single agent with multiple servers 
- Spin up one stack and "attach" multiple single-server-agents

Thank you!

**Comment by drupaladmin** [comment]
Added documentation on how to run multiple agents on the EC2 instance using Docker Compose https://docs.releem.com/releem-agent/installation-guides/cloud-managed-aws-rds-automatic-installation
